### ISSUE TITLE
Accessibility features

### ISSUE THREAD

**Issue by theRealCarneiro** [theRealCarneiro/pulsemeeter#135]
Pulsemeeter currently lacks accessibility features. There are no screen reader labels, keyboard navigation support, or alternative input methods implemented at this stage.

- [ ] Screen reader friendly ( #110)
   - [x] Labels
   - [x] Mnemonics
   - [ ] Translations #139 
- [ ] Keyboard navigation (#110)
   - [x] Tab and arrow keys navigation
   - [x] Confirm keys (return/enter and space) actually interact with widgets 
- [ ] Help menu
- [ ] Customizable shortcuts
- [ ] Real testing by users who rely on screen readers and keyboard navigation
